src/slider/slider.scss

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$thumb-height: $iui-m;  // 16px
+$track-height: $iui-xs;  // 4px
+$tick-height: $iui-baseline; // 11px
+
+@mixin iui-slider-component-container {
+  display: flex;
+
+  .iui-slider-min,
+  .iui-slider-max {
+    @include iui-font-family;
+    user-select: all;
+    margin-top: $iui-xxs;
+
+    .iui-icon {
+      @include iui-icons-default;
+      margin-top: $iui-xs;
+    }
+  }
+
+  .iui-slider-min {
+    margin-right: $iui-s;
+    text-align: right;
+  }
+
+  .iui-slider-max {
+    margin-left: $iui-s;
+  }
+
+  &.iui-disabled {
+    @include iui-slider-disabled;
+  }
+}
+
+@mixin iui-slider-container {
+  position: relative;
+  z-index: 1; // above ticks
+  flex-grow: 1;
+  pointer-events: all;
+  cursor: pointer;
+
+  .iui-tooltip {
+    display: none;
+  }
+
+  .iui-slider-rail {
+    pointer-events: all;
```

Comment:
any specific reason for this one here?

Reply:
I probably added it at some point to ensure the pointer was displayed. If it is no longer needed, it can be removed.